src/server/repository/public/user/update.ts

```diff
@@ -26,18 +27,15 @@ export const update = async (
   )
 
   if (profilePicture) {
-    const {
-      profilePicture: { originalname, buffer },
-    } = props
+    const createdFile = await FileRepository.create({ file: profilePicture }, client)
```

Comment:
Sure, updated here and cleaned the logic a bit to avoid double query:

https://github.com/openforis/fra-platform/pull/3819/commits/44159ced058a85581fe9686905dbee828046a49f